gematria/datasets/extract_bbs_from_obj_test.py

```diff
@@ -0,0 +1,11 @@
+from lit.main import main
```

Comment:
Nit: The Google Python style does not allow direct imports of classes and functions.

Reply:
Fixed.